src/components/MyButton.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <button class="button is-primary" @click="onClick">
+    <!-- @slot default inner button content -->
+    <slot></slot>
+  </button>
+</template>
+
+<script>
+export default {
+  name: "my-button",
```

Comment:
[Change]: Should we name this ButtonBase?

Reply:
this is the default button from vue and we should delete it actually haha, I will just make some placeholder and we will do the refactor of file structure and file naming 😚😚